src/services/order/order.service.ts

```diff
@@ -58,7 +56,7 @@ export class OrderService {
     await Promise.all([
```

Comment:
The await is not necessary inside the array. PromiseAll is useful when you need to send multiple transactions and you don't need a sequence.

Reply:
Yeah that was a copy/paste mistake. Thanks for catching that!